### DISCUSSION TITLE
[Feedback Wanted] Feeds are sorted in reverse order - bug fix shipped 🚀

### DISCUSSION THREAD

**Discussion by ebndev** [community/community#146771] [Other Feature Feedback, Questions, & Ideas]
Hi Community 🌟 

Thank you all for your posts and comments on the recent bug where the Feed was sorting in reverse order. Our Feed team pushed an update to address the issue on December 8th, 2024.

If you have further issues with this or additional details about your experience, please comment directly in this thread so we can effectively monitor the changes.

Please note: this bug is related to the larger initiative of [Reverting back to chronological sorting in the Feed](https://github.com/orgs/community/discussions/141611). Any questions or feedback regarding that should be posted in the linked discussion.

We appreciate you all for helping us continuously improve our platform!

**Comment by Igorgro** [comment]
There still problem with no releases are shown despite of selected filters. It was reported in multiple threads but didn't receive any official response. Will this problem be addressed?

**Reply by ebndev** [reply]
Hi @Igorgro,

I'm working with the Feed team to consolidate feedback in the category and will respond as soon as an update is available.

Thank you for your patience as we work toward a solution!

**Reply by OneSeven** [reply]
> Hi @Igorgro,
> 
> I'm working with the Feed team to consolidate feedback in the category and will respond as soon as an update is available.
> 
> Thank you for your patience as we work toward a solution!

I have had this problem for a long time. No matter how you switch the filter, the feed does not display any content. It has seriously affected daily use. Please speed up the solution!

**Reply by KieranP** [reply]
@Igorgro Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful, feel free to star the repo :-)

**Comment by shinybrightstar** [comment]
Hi Community 🌟

Thank you for your posts and comments regarding the recent bug with the Feed sorting in reverse order. Also, data retention has been updated from 90 days to 30 days. Our Feed team released an update on January 30th, 2025, to fix these issues.
Also, we’re working on resolving the “releases not showing” problem, which will be addressed by the end of February 2025.

If you’re still experiencing issues or have more details to share about your experience, please comment directly in this thread so we can monitor the changes effectively.

You can find more info [here](https://github.blog/changelog/2024-11-08-upcoming-changes-to-data-retention-for-events-api-atom-feed-timeline-and-dashboard-feed-features/)

We appreciate you all for helping us continuously improve our platform!
